### PR TITLE
[WIP] Convert to Swift 3.0 ✨✨

### PR DESCRIPTION
**Pull request by elland**
- Broke a test, because of new colour space handling.

### CODE REVIEW

File: Demo.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -124,6 +124,9 @@
 				TargetAttributes = {
 					146D72AB1AB782920058798C = {
 						CreatedOnToolsVersion = 6.2;
+						DevelopmentTeam = 29X3GG489T;
+						DevelopmentTeamName = "Bakken & Bæck AS";
```
Comment (3lvis) [inline Demo.xcodeproj/project.pbxproj]: 👀 would you mind removing that?

Comment (elland) [inline Demo.xcodeproj/project.pbxproj]: 👀 New xcode being sneaky?

**Comment by 3lvis** [comment]
Added https://github.com/3lvis/Hex/commit/b8978681846a1bfe45dac422eba4f6fd9b7b3e21

🎉

**Comment by 3lvis** [comment]
Haven't been able to figure out how to fix the `isEqual` method, though.

**Pull request closed without merging**